checker/check-response-optional-property-updated.go

```diff
@@ -32,25 +32,52 @@ func ResponseOptionalPropertyRemovedCheck(diffReport *diff.Diff, operationsSourc
 
 				modifiedMediaTypes := responseDiff.ContentDiff.MediaTypeModified
 				for _, mediaTypeDiff := range modifiedMediaTypes {
+					comment := ""
 					CheckDeletedPropertiesDiff(
 						mediaTypeDiff.SchemaDiff,
 						func(propertyPath string, propertyName string, propertyItem *openapi3.Schema, parent *diff.SchemaDiff) {
+							level := WARN
 							if propertyItem.WriteOnly {
-								return
+								level = INFO
```

Comment:
Here, I'm making sure write-only properties will also be captured in the changelog. I don't think it's needed to create a separate text for it so Im adding it as a comment. Let me know if it makes sense or if we prefer to add a new checker id "response-optional-write-only-property-removed".

cc @tibulca @reuvenharrison

Reply:
Looking good, returning the INFO level will filter out this entry when running -check-breaking but will keep it when running -changelog.
Regarding the comment vs id, my preference would be to have a separate `response-optional-write-only-property-removed` id. Easier to (programmatically) process an entry with a specific id, than to check a comment.